introduction/features.md

```diff
@@ -1,19 +1,22 @@
 ## Features
 
-* Fast
-* [Typo tolerance](/advanced_guides/typotolerance.md#typo-tolerance) (understands typo and spelling mistakes)
-* Return whole document
-* Crop, snippets, filters, highlight in search queries
-* Search inside chosen attributes
-* Highlighted results
-* Prefix-search
-* Configurable Stop-words
-* Configurable Synonyms
-* Asynchronous action on POST, PUT, and DELETE
-* RESTfull API
-* Supported Latin and Kanji [tokenizer](https://github.com/meilisearch/MeiliDB/blob/dc5c42821e1340e96cb90a3da472264624a26326/meilidb-tokenizer/src/lib.rs) 
-* Bucket sort with 6 default rankings criteria
-* Custom criteria (ranking, distinct, ...) in schema definition
-<!-- Schemaless possibility -->
+Our feature list with extended explanations and links to the documentation.
 
 
+* **Instant Search** (answers < 50ms): Priority on fast answers for smooth search experience.
+* **Search as you type** (*prefix search*): Results are shown as you type. To achieve this, we use a [prefix-search](/advanced_guides/prefix.html#prefix-search).
+* [Typo tolerance](/advanced_guides/typotolerance.md#typo-tolerance): Understands typo and spelling mistakes.
+* [Tokenization](https://en.wikipedia.org/wiki/Lexical_analysis#Tokenization) in English, kanji and latin based languages.  
+* Ability to create [synonyms](/advanced_guides/synonyms.md) for a better search experience.   
+* [Easy to install, deploy and maintain](/introduction/quickstart.html#quick-start)
+* **Return the whole document**: The entire document is returned upon search.
+* **Highly customizable search and indexation**:
+    - [Filters](/advanced_guides/search_parameters.md#filters): Filter in search
+    - [Highlight](/advanced_guides/search_parameters.md#attributes-to-highlight): Highlighted search results in documents
+    - [Maximum response time](/advanced_guides/search_parameters.md#attributes-to-highlight): Never will a response take more time to arrive
```

Comment:
I am not this feature exists today

Reply:
Well, it exists, but it doesn't work. I'll come back to this request once we know when it will be fixed